extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -310,8 +323,29 @@ export class SchemaCompareResult {
 		});
 	}
 
+	private createOptionsButton(view: azdata.ModelView) {
+		this.optionsButton = view.modelBuilder.button().withProperties({
+			label: localize('schemaCompare.optionsButton', 'Options'),
+			iconPath: {
+				light: path.join(__dirname, 'media', 'options.svg'),
+				dark: path.join(__dirname, 'media', 'options_reverse.svg')
+			},
```

Comment:
add tooltip(title property) for optionsButton

Reply:
done